<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartSearchForm</class>
 <widget class="QWidget" name="StartSearchForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Destruction magic</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Min thunder count</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Starting seed</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Number of seeds</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Min dragon count</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Equip</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Field spell</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="startBox"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="numbSeedsBox"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="dragBox"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="thundBox"/>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="destMagicBox"/>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="equipBox"/>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="fieldBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Other required cards</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="specificsBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="genDecksBtn">
       <property name="text">
        <string>Find decks</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Valid decks</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="goodDecksBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Deck listing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="deckBox">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
